communities/epfl-hackers/community-assessment.md

```diff
@@ -0,0 +1,87 @@
+### Definition
+- How would you define your community? Try to be open minded, but here are some examples of ways you could define your community: 
+  - Students interested in working on cool tech projects outside of academia at EPFL.
+  
+### Size
+- How many people are currently in your community?
+  - Hard to tell as there is no official club or society, but I would estimate we are around 5 people.
```

Comment:
I totally agree! For the hackathon I organized last semester I worked with an association called Clic which organizes social events for computer science students at EPFL and so it was branded a "Clic event" which helped with publicity on campus. But we're trying to see if we can form a new association for the next academic year which is more focused.